src/extension-runners/index.js

```diff
@@ -232,13 +232,14 @@ export type WatcherCreatorParams = {|
   onSourceChange?: OnSourceChangeFn,
   ignoreFiles?: Array<string>,
   createFileFilter?: FileFilterCreatorFn,
+  reloadDelay: Number
```

Comment:
Add a trailing comma (also in the rest of this PR where applicable).

The comma is optional for the last property of an object literal, but we still use it anyway, because doing so makes it possible to add a new property without having to change the previous line.